### ISSUE TITLE
utilisation smartphone en mode 'application': pas de d'affichage POI

### ISSUE THREAD

**Issue by u4y0u** [laem/futureco#239]
Lors de l'utilisation en mode application, l'affichage des POI ne se fait pas du tout et il est impossible de 'raffraichir' l'écran comme dans le mode web.
Voici des copies d'écran d'exemple de passage aux POI restaurant et caféou l'on ne voie aucun POI en mode App alors que le même lieu affiché dans un browser fait parfaitement ressortir les POI.
Bizzarement les POI apparaissent dans l'application si l'on lance l'application à partir d'une fenêtre web (voir dernieres copies d'écran)

mode application: POI restaurant (rien....)
![Screenshot_2024-04-03-10-58-57-680_com android chrome](https://github.com/laem/futureco/assets/54479065/1b4e3f62-9606-456d-b1b9-873846a78e03)

mode application : passage en POI 'café' avec appuie sur le bouton POI (toujours rien)
![Screenshot_2024-04-03-10-59-30-995_com android chrome](https://github.com/laem/futureco/assets/54479065/9e7f5ebe-ca8d-4323-a4fc-b643e855d41a)

passage en mode browser web: POI restaurant (Ok)
![Screenshot_2024-04-03-11-00-59-969_org mozilla firefox](https://github.com/laem/futureco/assets/54479065/b7fecb43-f27f-4cf1-8101-d40359a364d2)

passage en POI café toujours via browser (toujours OK)
![Screenshot_2024-04-03-11-01-18-076_org mozilla firefox](https://github.com/laem/futureco/assets/54479065/5e6d7ed1-3bd9-4782-ac91-8113c0a4d980)

passage en mode 'ouvrir dans une application...' à partir de la fenêtre web...
![Screenshot_2024-04-03-11-15-05-134_org mozilla firefox](https://github.com/laem/futureco/assets/54479065/c19e0883-32d1-4148-b46f-a5845662c2a2)

...apparition des POI dans l'application !
![Screenshot_2024-04-03-11-15-14-853_org chromium webapk afa973698c37093bc_v2](https://github.com/laem/futureco/assets/54479065/026295ef-6b3a-4725-aefc-8edced6c1f0a)

**Comment by laem** [comment]
Dingue ! On dirait que la requête est bloquée. Pas facile à investiguer ça. 
J'avoue personnellement que l'usage en mode application me semble compliqué. Le simple fait de pas pouvoir recharger, c'est une des bases du Web qu'ils ont enlevé... Ça demande beaucoup de travail en plus. Ce serait bien de s'y mettre clairement mais là on n'y est pas encore.

**Comment by u4y0u** [comment]
Effectivement, ce n'est pas très génant car on peut l'utiliser en mode web. C'est juste déroutant pour un nouvel utilisateur qui n'a aucune réaction lorsqu'il appuie sur les boutons.

**Comment by laem** [comment]
Salut ! Je crois avoir résolu le souci. Avec une refacto des fichiers que j'avais faite, j'avais détruit le cartes.app/manifest.webmanifest. 

C'est résolu. Peux-tu retester ?

**Comment by Jean-BaptisteC** [comment]
C'est tout bon, ça peut être clôturé
